src/wp-includes/template.php

```diff
@@ -704,11 +704,14 @@ function locate_template( $template_names, $load = false, $load_once = true, $ar
 		if ( ! $template_name ) {
 			continue;
 		}
-		if ( file_exists( STYLESHEETPATH . '/' . $template_name ) ) {
-			$located = STYLESHEETPATH . '/' . $template_name;
+
+		$stylesheet_path = get_stylesheet_directory();
+		$template_path   = get_template_directory();
```

Comment:
Could you please conduct a performance benchmark after the changes? It would be great if you could add at least 10 posts to your setup before conducting the benchmark. Thanks!